ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -70,6 +71,21 @@ private[commands] class CommandTracker[Context]
 
     val logic: TimerGraphStageLogic = new TimerGraphStageLogic(shape) {
 
+      val timeout_detection = "timeout-detection"
```

Comment:
This doesn't seem to follow Scala naming guidelines:
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods